helm/conjur-config-cluster-prep/templates/tests/_helm_test.bats.txt

```diff
@@ -50,7 +50,7 @@ function conjur_access_token_exists() {
   run conjur_is_reachable
   if [ "$status" -ne 0 ]; then
     display_error "The 'conjur.applianceURL' chart value is set to\n" \
-                  "$conjurApplianceURL. This is not reachable via 'curl -k'"
+                  "$conjurApplianceUrl. This is not reachable via 'curl -k'"
```

Comment:
I think we should consider updating the messaging to something that gives some explanation for the reachability  
```
This is not reachable outside the Kubernetes network, so running 'curl -k ...' on your local machine probably won't work!
```

Reply:
It could be internal to the K8s network, and just a config error. That's what we are looking to detect.

Reply:
To add to that... the `curl -k <Conjur-appliance-URL>` is always executed from a Pod inside the K8s cluster because this is a Helm test, but the destination Conjur appliance URL can be an internal K8s address or an external-to-k8s address.